### PR TITLE
[JENKINS-33345] Add ability to define list of image pull secrets for pod template

### PR DESCRIPTION
**Pull request by moserke**
Solution for https://issues.jenkins-ci.org/browse/JENKINS-33345

### CODE REVIEW

File: src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodImagePullSecret/config.jelly [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+THE SOFTWARE.
+-->
+
+<!--
+  Config page
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+blardy blog
```
Comment (carlossg) [inline src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodImagePullSecret/config.jelly]: needs to be removed?

Comment (moserke) [inline src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodImagePullSecret/config.jelly]: oh geez.  yeah.  pushed a change without that in there.

**Comment by Rastusik** [comment]
@carlossg would it be possible to merge this request? I'm quite stuck without it ... or is there a guide somewhere on how to manually build the plugin? thank you

**Comment by carlossg** [comment]
Merged in 983861f

**Comment by Rastusik** [comment]
thank you! one more question - is there a plan for the next release? a date?

**Comment by carlossg** [comment]
I'll probably cut 0.8 now

**Comment by Rastusik** [comment]
that's great, you saved me a couple of hours, thank you very much! :) 👍

**Comment by w0mbat** [comment]
This sounds great - thanks everyone!
We did a workaround in between with a patch for the default service account, but its awesome that we could now use different secrets for each slave and also have this functionality without patching the serviceaccount.

**Pull request closed without merging**